.pipelines/templates/log-template.yaml

```diff
@@ -0,0 +1,222 @@
+# -- Captures --
+# CNS, CNI, and Cilium Logs
+# CNS, CNI, and Cilium State files
+# Daemonset and Deployment Images
+# kube-system namespace logs
+# Non-ready pods on failure
+# -- Controled by --
+# CNI and OS | ${{ parameters.cni }} and ${{ parameters.os }}
+# -- Generates --
+# Logs on a per-node basis
+# Outputs a singluar unique artifact per template call | ${{ parameters.clusterName }}_${{ parameters.logType }}_Attempt_#$(System.StageAttempt)
+# -- Intent --
+# Provide through debugging information to understand why CNI test scenarios are failing without having to blindly reproduce
+
+parameters:
+  clusterName: ""
+  logType: "failure"
+  os: ""
+  cni: ""
+
+steps:
+  - task: AzureCLI@1
+    inputs:
+      azureSubscription: $(BUILD_VALIDATIONS_SERVICE_CONNECTION)
+      scriptLocation: "inlineScript"
+      scriptType: "bash"
+      addSpnToEnvironment: true
+      inlineScript: |
+          set -e
+          make -C ./hack/aks set-kubeconf AZCLI=az CLUSTER=${{ parameters.clusterName }}
+
+          acnLogs=$(System.DefaultWorkingDirectory)/${{ parameters.clusterName }}_${{ parameters.logType }}_Attempt_#$(System.StageAttempt)
+          mkdir -p $acnLogs
+          echo Root Directory created: $acnLogs
+          echo "##vso[task.setvariable variable=acnLogs]$acnLogs"
+
+          kubectl get pods -n kube-system -owide
```

Comment:
Good to have a node status too ?

Reply:
As a separate step similar to `Daemonset/Deployment Images?` 

I would want it to be verbose with describe so we can see taints, events, pods assigned ect.

Reply:
Either way.